src/screens/faq/styles.tsx

```diff
@@ -0,0 +1,100 @@
+import { makeStyles } from '@material-ui/styles';
+import { useTheme } from '@material-ui/core/styles';
+import { getMinMediaQuery } from '@styles';
+
+export const useGetStyles = () => {
+  const theme: any = useTheme();
+  const useStyles = makeStyles({
+    root: {
+      padding: '1rem 1rem 2.5rem',
+    },
+    title: {
+      fontSize: '1.375rem',
+      fontWeight: 500,
+      margin: '0',
+      marginBottom: '0.5rem',
+      marginTop: '1rem',
+      lineHeight: '1.375rem',
+      color: '#000000',
```

Comment:
remember to handle dark theme~
![image](https://user-images.githubusercontent.com/42913823/96405298-8596e400-120f-11eb-8bf7-a3a866efb5aa.png)

Reply:
done